frontend/views/containers/access/LoginForm.vue

```diff
@@ -93,12 +97,12 @@ export default ({
         })
         await this.postSubmit()
         this.$emit('login-status', 'success')
-
-        requestNotificationPermission().catch(e => console.error('[SignupForm.vue] Error requesting notification permission', e))
```

Comment:
This is here for a reason. In most browsers, the notification permission needs to be requested as a direct result of user interaction. Your change will make it more likely for the request to fail.